src/chaiPreferContainsToIndexOfRule.ts

```diff
@@ -17,7 +17,7 @@ export class Rule extends Lint.Rules.AbstractRule {
         ruleName: 'chai-prefer-contains-to-index-of',
         type: 'maintainability',
         description: 'Avoid Chai assertions that invoke indexOf and compare for a -1 result.',
-        options: null,
+        options: undefined,
```

Comment:
This one always confused me. TSLint itself describes the rule's options as:

```typescript
Schema of the options the rule accepts.
The first boolean for whether the rule is enabled or not is already implied.
This field describes the options after that boolean.
If null, this rule has no options and is not configurable.
```

https://github.com/palantir/tslint/blob/237fd905a9fb3c9ca9325d2da6e8145e4a44fe51/src/language/rule/rule.ts#L64

...so we should probably default this to `null`.

Reply:
Ah, yes, you're absolutely right. I had a look at that comment as well as how the options are used in TSLint before I made the changes and it looked like it shouldn't matter if undefined is used, but of course the _one_ place I didn't look at was what TSLint uses for their own rules, and they use null.